yt/yt/server/master/security_server/access_log.h

```diff
@@ -92,4 +88,6 @@ bool IsAccessLoggedType(const NCypressClient::EObjectType type);
 
 ////////////////////////////////////////////////////////////////////////////////
 
+bool IsAccessLogEnabled(NCellMaster::TBootstrap* bootstrap);
```

Comment:
Namespace should always terminate with "separator line" aka "otbivka", so either you should but a new line consisting of ///...//, or move this declaration together with some of the declarations above.